docs/intro.md

```diff
@@ -0,0 +1,39 @@
+# Hydra-genetics qc module
+The qc module is a collection of qc programs that performs analysis on the aligned bam-files as well as fastq-files and then collects these qc-measures in a html-report with MultiQC.
+
+## QC programs
+
+| Program | Input | Output |
+|-|-|-|
+| [fastqQC](https://github.com/s-andrews/FastQC) | fastq | Sequencing report |
+| [GATK](https://gatk.broadinstitute.org/hc/en-us) | pileup | Sample contamination |
+| [mosdepth](https://github.com/brentp/mosdepth) | bam | Coverage |
+| [picard](https://broadinstitute.github.io/picard/) | | Alignment statistics, gc bias, panel statistics, insert size |
+| [rseqc](https://rseqc.sourceforge.net/) | | RNA gene coverage and inner distance |
+| [samtools](http://www.htslib.org/doc/samtools-stats.html) | _ _ | Alignment statistics |
+| [peddy](http://www.htslib.org/doc/samtools-stats.html) | vcf | Sex and relatedness checks |
+| [multiQC](https://github.com/ewels/MultiQC) | QC-files | Summary html-report |
+
```

Comment:
Verifybamid missing

Reply:
Addressing this and all the other readthedocs issues in https://github.com/hydra-genetics/qc/pull/117